docs/deploy.rst

```diff
@@ -0,0 +1,229 @@
+    +--------------------------------------+----------------+-------------+
+    | id                                   | name           |   plan_type |
+    |--------------------------------------+----------------+-------------|
+    | 59c7961b-33a2-4478-9e98-2a6b3f54f60e | MyOrganization |           1 |
+    +--------------------------------------+----------------+-------------+
+
+This will output a list of all organizations that you are a member of. The
+command's output includes the organization's org-ids, names, and plan types.
+
+Create a project
+================
+
+Set a region in Croud
+=====================
+
+First of all, set a region in Croud. The ``region`` setting determines in which
+geographical location your data is stored. It also specifies which CrateDB
+products, such as clusters, are available to you.
+
+The following regions are available: Azure East-US, Azure West-Europe, Bregenz.
+
+These are your possible choices for the ``region`` flag in the ``croud config``
+command:
+
+- bregenz.a1
+- westeurope.azure
+- eastus2.azure
+- eastus.azure
+
+This tutorial uses the region ``bregenz.a1``.
+
+Run the `` croud config`` command and pass in ``region`` to set a region, like
+so:
+
+  .. code-block:: console
+
+     sh$ croud config set --region bregenz.a1
+     ==> Info: Region switched to bregenz.a1
+
+Projects contain CrateDB products such as clusters and are created in specific
+regions. A CrateDB product that you deploy in a specific project is tied to its
+region.
+
+To create a project run the  `croud projects command`_ and pass in ``--name`` to
+specify the project name, as well as ``org-id`` to specify your organization as
+the organization to create the project in:
+
+.. TIP::
+
+  Use the ID of the organization you created in the previous step.
+
+.. code-block:: console
+
+    sh$ croud projects create --name YourProjectName \
+        --org-id 59c7961b-33a2-4478-9e98-2a6b3f54f60e
+
+You have created a project in your organization.
+
+.. SEEALSO::
+
+  Look up the `Croud CLI section`_ for help using the ``croud organizations`` or
+  ``croud projects`` commands.
+
+Look for available products
+===========================
+
+Now that you've set a region, it's time to look for available products in that
+region.
+
+Run the ``croud products`` command and pass in ``region`` to look for available
+products:
+
+.. code-block:: console
+
+   sh$ croud products list --region bregenz.a1
+   +-------------------------------------------------------------+----------+-------------------+---------------------------+--------+
+   | description                                                 | kind     | name              | scale_summary             | tier   |
+   |-------------------------------------------------------------+----------+-------------------+---------------------------+--------|
+   | A CrateDB cluster suited for general purpose workloads      | cluster  | cratedb.az-gp1    | 0 - 0 (1 - 1 nodes)       | xxs    |
```

Comment:
this tier is not available in prod.

I think the docs should only list stuff available in prod

Reply:
Running `croud products list --region eastus.azure` the xxs tier is available for me.

Reply:
That's because you're running against the development environment. You can use `--env prod` to ensure you're running against the production environment.